book/src/07_join.md

```diff
@@ -1,7 +1,7 @@
 # Joining components
 
 In the last chapter, we learned how to access resources using `SystemData`.
-To access our components with it, we can just fetch a `ReadStorage` and use
+To access our components with it, we can just get a `ReadStorage` and use
 `Storage::get` to get the component associated to an entity. This works quite
```

Comment:
It's kinda unfortunate that there is three different uses of 'get' here, but it's not that big of a problem.